rviz_default_plugins/include/rviz_default_plugins/displays/camera/camera_display.hpp

```diff
@@ -174,6 +174,10 @@ private Q_SLOTS:
     ImageDimensions dimensions,
     const Ogre::Vector2 & zoom) const;
 
+  Ogre::Vector4 calculateScreenCorners(
+    sensor_msgs::msg::CameraInfo::ConstSharedPtr info,
+    const Ogre::Vector2 & zoom) const;
+
```

Comment:
Since the implementation does not rely on any member variables, I think it would be better to declare and define this as a static free function in the cpp file. This helps keep the API surface area smaller.